### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by CuDecker** [CuDecker/MI-449-SS17-740-html-atom-and-basic-elements-JmL5o8#1]
@egillespie Can you take a look at this? It's [hosted here](https://CuDecker.github.io/MI-449-SS17-740-html-atom-and-basic-elements-JmL5o8/) and meets the following criteria:

- [x] Each of the following elements is used appropriately, at least once: `h1`, `h2`, `p`, `img`, `li`, `a`
- [x] At least one of the following elements is used: `ol`, `ul`
- [x] At least one external image is used and links to the page it came from

The links to the RF picture is in the Alt text. Alternatively you can also just click the picture. The link should work

**Comment by egillespie** [comment]
@CuDecker It looks like you haven't pushed your changes to the `gh-pages` branch ([this link](https://cudecker.github.io/MI-449-SS17-740-html-atom-and-basic-elements-JmL5o8/) should show your work).

In a terminal, change to your project directory and run the following command:

```
git push origin master:gh-pages
```

When that's done and you can see [your site](https://cudecker.github.io/MI-449-SS17-740-html-atom-and-basic-elements-JmL5o8/), comment here and I'll take another look.

Thanks!

**Comment by CuDecker** [comment]
So git push origin master:gh-pages is what I need to do to actually activate the github pages? 

that link works now, thanks!

**Comment by egillespie** [comment]
@CuDecker Exactly! Not all of our projects use GitHub Pages, though, so be careful to use the correct approach when uploading the live version of your project.

Everything looks great now! :shipit:
